code/game/gamemodes/miniantags/guardian/guardian.dm

```diff
@@ -165,23 +165,21 @@
 //Manifest, Recall, Communicate
 
 /mob/living/simple_animal/hostile/guardian/proc/Manifest()
-	if(cooldown > world.time)
+	if(!summoner || !COOLDOWN_FINISHED(src, manifest_cooldown))
 		return
-	if(!summoner) return
 	if(loc == summoner)
 		forceMove(get_turf(summoner))
 		new /obj/effect/temp_visual/guardian/phase(loc)
 		reset_perspective()
-		cooldown = world.time + 30
+		COOLDOWN_START(src, manifest_cooldown, 3 SECONDS)
 
 /mob/living/simple_animal/hostile/guardian/proc/Recall(forced = FALSE)
-	if(!summoner || loc == summoner || (cooldown > world.time && !forced))
+	if(!summoner || loc == summoner || (!COOLDOWN_FINISHED(src, manifest_cooldown) && !forced))
 		return
-	if(!summoner) return
```

Comment:
this for me seems to be a needed last ditch fallback i could be wrong.

Reply:
It may have originally been intended that way, but as there's another `summoner` check already two lines above, this one will never actually fail.
https://github.com/ParadiseSS13/Paradise/blob/0ce6bba543b0dc8c776f8a39913424ace071a644/code/game/gamemodes/miniantags/guardian/guardian.dm#L177-L184
I've not been playing for long enough to know for sure, but if there was an "Are you sure you want to recall?" input between these two lines in the past, then that would explain the extra check. Since the `summoner` could have been deleted in the wait time.
Either way, since there's currently no sleeps or anything between L178 and L181, the check won't ever be nessecary.